From 94d6e6fe0af969fb44b14c2c869655d2e3c253f1 Mon Sep 17 00:00:00 2001
From: Jimmy Xiang <jxiang@cloudera.com>
Date: Thu, 2 May 2013 13:27:41 -0700
Subject: [PATCH 86/96] Revert "HBASE-8389 HBASE-8354 DDoSes Namenode with lease recovery requests (Varun and Ted)"

Reason: Bug
Ref: CDH-11864

This reverts commit 9077b5a52c3eeade554bf0cbfc648068d00fc891.
---
 .../org/apache/hadoop/hbase/util/FSHDFSUtils.java  |    9 ++-------
 1 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/util/FSHDFSUtils.java b/src/main/java/org/apache/hadoop/hbase/util/FSHDFSUtils.java
index ea15e10..968e3c4 100644
--- a/src/main/java/org/apache/hadoop/hbase/util/FSHDFSUtils.java
+++ b/src/main/java/org/apache/hadoop/hbase/util/FSHDFSUtils.java
@@ -73,9 +73,6 @@ public class FSHDFSUtils extends FSUtils{
     long recoveryTimeout = conf.getInt("hbase.lease.recovery.timeout", 300000);
     // conf parameter passed from unit test, indicating whether fs.append() should be triggered
     boolean triggerAppend = conf.getBoolean(TEST_TRIGGER_DFS_APPEND, false);
-    // retrying lease recovery may preempt pending lease recovery; default to waiting for 4 seconds
-    // after calling recoverLease
-    int waitingPeriod = conf.getInt("hbase.lease.recovery.waiting.period", 4000);
     Exception ex = null;
     while (!recovered) {
       try {
@@ -130,15 +127,13 @@ public class FSHDFSUtils extends FSUtils{
       }
 
       try {
-        Thread.sleep(waitingPeriod);
+        Thread.sleep(1000);
       } catch (InterruptedException ie) {
         InterruptedIOException iioe = new InterruptedIOException();
         iioe.initCause(ie);
         throw iioe;
       }
-      // we keep original behavior without retrying lease recovery
-      break;
     }
-    LOG.info("Finished lease recovery attempt for " + p);
+    LOG.info("Finished lease recover attempt for " + p);
   }
 }
-- 
1.7.0.4

